internal/apiserver/invoiceValidator_test.go

```diff
@@ -0,0 +1,15 @@
+package apiserver
```

Comment:
I would move this file to `invoiceValidator` package

I think it should be ok import it then in `main_test`